mismi-s3/src/Mismi/S3/Commands.hs

```diff
@@ -520,6 +533,10 @@ handle404 :: AWS a -> AWS (Maybe a)
 handle404 m = fmap Just m `catch` \ (e :: Error) ->
   if e ^? httpStatus == Just status404 then return Nothing else throwM e
 
+handle301 :: AWS a -> AWS (Maybe a)
+handle301 m = fmap Just m `catch` \ (e :: Error) ->
+  if e ^? httpStatus == Just status301 then return Nothing else throwM e
+
```

Comment:
Probably naive question, but why do we handle 301 (Moved Permanently) error? I've seen it in the situation when the wrong `Region` was used for existing bucket. This isn't such case, is it?